feature-libs/product-configurator/rulebased/components/attribute/quantity/configurator-attribute-quantity.component.ts

```diff
@@ -40,22 +40,29 @@ export class ConfiguratorAttributeQuantityComponent
 
   ngOnInit() {
     this.quantity.setValue(this.quantityOptions?.initialQuantity?.quantity);
-
-    if (this.quantityOptions?.disableQuantityActions) {
-      this.quantity.disable();
-    }
+    const sub = this.quantityOptions.disableQuantityActions?.subscribe(
+      (disable) => {
+        if (disable) {
+          this.quantity.disable();
+        } else {
+          this.quantity.enable();
+        }
+      }
+    );
+    if (sub) this.managedSubscriptions.add(sub);
 
     const debounceQuantity = this.quantity.valueChanges.pipe(
       debounce(() =>
         timer(this.config?.rulebasedConfigurator?.quantityDebounceTime)
       )
     );
```

Comment:
yes there was missing something - actually it seems that the stepper itself is emitting a quantity change event in case it gets enabled, even if quantity it self was not changed at all.
That leads to an endless loop. as the quantity update will disabled/enable the stepper and hence the next quantity change is emitted and so on.
So i now unsubscribe on disable and re-subscribe on enable.  Maybe this could also be handle by a filter, but i had troubles finding one.

Reply:
I don't get that, why not simply take(1)?
I tried that, for me it's working fine. Or did you face an issue with it?